src/views/Main/Header/LoginButton/index.js

```diff
@@ -0,0 +1,6 @@
+import { withTranslation } from 'react-i18next';
+import { connectAuthProvider } from '@terralego/core/modules/Auth';
+
+import LoginButton from './LoginButton';
+
+export default withTranslation()(connectAuthProvider('authenticated', 'logoutAction')(LoginButton));
```

Comment:
Maybe it would be interesting to grouping all HOC here (including `widthDeviceSize()` in ` src/views/Main/Header/LoginButton/LoginButton.js`)